infra/terraform/gcp/modules/ComputeInstance/airqo-dev-k8s-controller.tf

```diff
@@ -4,9 +4,11 @@ resource "google_compute_instance" "airqo_dev_k8s_controller" {
   zone    = var.zone["b"]
 
   machine_type = "e2-standard-2"
+  
+  allow_stopping_for_update = true
```

Comment:
[allow stopping to update](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/a67cd572f2c7b6c301368f0224d36d6aa18a812a)